gradle/libs.versions.toml

```
@@ -226,6 +227,7 @@ androidx-test-runner = { module = "androidx.test:runner", version.ref = "test" }
 # Tracks
 automattic-crashlogging = { module = "com.automattic.tracks:crashlogging", version.ref = "tracks" }
 automattic-tracks = { module = "com.automattic:Automattic-Tracks-Android", version.ref = "tracks" }
+automattic-explat = { module = "com.automattic.tracks:experimentation", version.ref = "explat" }
```

```suggestion
automattic-explat = { module = "com.automattic.tracks:experimentation", version.ref = "tracks" }
```

All the "tracks" modules are shipped from the same repository and use the same version. To reduce complexity, I propose unifying it.